x-pack/plugin/relevance-search/src/main/java/org/elasticsearch/xpack/relevancesearch/query/CurationQueryRewriter.java

```diff
@@ -39,12 +42,18 @@ public QueryBuilder rewriteQuery(
         RelevanceMatchQueryBuilder relevanceMatchQuery,
         SearchExecutionContext context
     ) {
-        CurationSettings curationSettings = getSettings(relevanceMatchQuery, context);
+        CurationsGroup curationGroup = getSettings(relevanceMatchQuery, context);
+
+        if (curationGroup == null) {
```

Comment:
Is it worth also returning `baseQuery` here if `curationGroup` doesn't have any curations (empty list)?

Reply:
I think the next code block will handle that (there won't be any matching condition)